pants.remote.toml

```diff
@@ -0,0 +1,32 @@
+[GLOBAL]
+remote_execution = true
+remote_execution_server = "build.toolchain.com:443"
+remote_store_server = "build.toolchain.com:443"
+remote_ca_certs_path = "/etc/ssl/certs/ca-certificates.crt"
+remote_instance_name = "main"
+remote_execution_extra_platform_properties = [
+  # This allows network requests, e.g. to resolve dependencies with Pex.
+  "execution-policy=network",
+  "execution-policy=container-image=pants-remote-execution:5175b2b04aa895732ecdcb98af9481866472a0bf",
+]
+process_execution_remote_parallelism = 10
+
+[python-setup]
+interpreter_search_paths = [
+  # These are the interpreter paths we set up on the remote container.
+  "/pyenv-docker-build/versions/3.8.5/bin:/pyenv-docker-build/versions/3.7.8/bin:/pyenv-docker-build/versions/3.6.11/bin:/pyenv-docker-build/versions/2.7.18/bin",
+]
+
+[pex]
+executable_search_paths = [
+  # These are the interpreter paths we set up on the remote container. We need to specify these
+  # because many of our tests for the Python backend need to discover Python interpreters.
+  "/pyenv-docker-build/versions/3.8.5/bin:/pyenv-docker-build/versions/3.7.8/bin:/pyenv-docker-build/versions/3.6.11/bin:/pyenv-docker-build/versions/2.7.18/bin",
```

Comment:
I don't think is applicable for the example repo. This is only so that Pants can run tests on the Python backend.

Reply:
Note from the container-image that we are using the Pants remote execution image. And these are the paths in that container.

Reply:
Yes, I agree. But I'm saying that we don't need to set this value here in `--executable-search-paths`. This option is used to set up the `$PATH` env var when running a process with Pex. We don't need Python discoverable on the $PATH afaict.

Pex will discover our interpreters instead thorugh `--python-setup-interpreter-search-paths`, which gets converted into `PEX_PYTHON_PATH`.

Normally, we don't expect end users to have Python interpreters in `--executable-search-paths`; that is a special thing we have to do in Pants because so many Pants tests run Python in the subprocesses spawned by Pants.

Reply:
I added it because the execution fails otherwise.

With this diff:

```
diff --git a/pants.remote.toml b/pants.remote.toml
index c839498..2e365b7 100644
--- a/pants.remote.toml
+++ b/pants.remote.toml
@@ -19,9 +19,6 @@ interpreter_search_paths = [

 [pex]
 executable_search_paths = [
-  # These are the interpreter paths we set up on the remote container. We need to specify these
-  # because many of our tests for the Python backend need to discover Python interpreters.
-  "/pyenv-docker-build/versions/3.8.5/bin:/pyenv-docker-build/versions/3.7.8/bin:/pyenv-docker-build/versions/3.6.11/bin:/pyenv-docker-build/versions/2.7.18/bin",
   # The remote container has binaries like `ld` and `gcc` in /usr/bin.
   "/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin",
 ]
```

You get this error:

```
  ProcessExecutionFailure: Process 'Building pytest.pex with 3 requirements: pytest-cov>=2.10.1,<2.11, pytest>=6.0.1,<6.1, zipp==2.1.0' failed with exit code 102.
stdout:

stderr:
/toolchain/operations/pex/.bootstrap/pex/pex_info.py:327: PEXWarning: PEX_ROOT is configured as /home/pantsbuild/.pex but that path is un-writeable, falling back to a temporary PEX_ROOT of /toolchain/operations/._tc_tmp/tmpXpuA2U which will hurt performance.
/toolchain/operations/._tc_tmp/tmpXpuA2U/unzipped_pexes/7d2ca8805b119a0fa474763a98c6191d28b169e0/.bootstrap/pex/variables.py:324: PEXWarning: PEX_ROOT is configured as /home/pantsbuild/.pex but that path is un-writeable, falling back to a temporary PEX_ROOT of /toolchain/operations/._tc_tmp/tmpmaCJlY which will hurt performance.
  "performance.".format(pex_root=pex_root, tmp_root=tmp_root)
/toolchain/operations/._tc_tmp/tmpXpuA2U/unzipped_pexes/7d2ca8805b119a0fa474763a98c6191d28b169e0/.deps/pex-2.1.14-py2.py3-none-any.whl/pex/variables.py:324: PEXWarning: PEX_ROOT is configured as /home/pantsbuild/.pex but that path is un-writeable, falling back to a temporary PEX_ROOT of /toolchain/operations/._tc_tmp/tmpmK2TS1 which will hurt performance.
  "performance.".format(pex_root=pex_root, tmp_root=tmp_root)
Could not find a compatible interpreter.

Examined the following interpreters:
  /pyenv-docker-build/versions/2.7.18/bin/python2.7 CPython==2.7.18
                                 /usr/bin/python2.7 CPython==2.7.12

None were compatible with the requested constraints:
  CPython>=3.7

Reply:
Oh, huh. That is not what we intended. Good to know. Thanks!